rites/roman1969/src/calendars/countries/brazil/index.ts

```diff
@@ -52,15 +59,36 @@ export class Brazil extends CalendarDef {
       titles: { append: [PatronTitle.PatronessOfBrazil] },
     },
 
-    anthony_of_saint_anne_galvao_priest: {
+    our_lady_of_mount_carmel: {
+      precedence: Precedences.ProperFeast_8f,
+      dateDef: { month: 7, date: 16 },
+    },
+
+    paulina_of_the_agonizing_heart_of_jesus_visintainer_virgin: {
       precedence: Precedences.ProperMemorial_11b,
-      dateDef: { month: 10, date: 25 },
+      dateDef: { month: 7, date: 9 },
     },
 
     roch_gonzalez_alphonsus_rodriguez_and_john_del_castillo_priests: {
       precedence: Precedences.ProperMemorial_11b,
       dateDef: { month: 11, date: 19 },
       martyrology: ['roch_gonzalez_priest', 'alphonsus_rodriguez_priest', 'john_del_castillo_priest'],
     },
+
+    rose_of_lima_virgin: {
+      precedence: Precedences.ProperFeast_8f,
+      dateDef: { month: 8, date: 23 },
+    },
   };
 }
+/*
+ Sources:
+  (1) Palavra do Senhor III - Lecionário para as missas dos Santos, dos comuns,
+      para diversas necessidades e votivas - Ed. Paulus (Lectionary for the Masses of Saints)
+      Respecting Upper and LowerCase (https://archive.org/details/lecionario-santoral-indice)
+  (2) Calendário Próprio do Brasil - CNBB (https://www.cnbb.org.br/missal-romano-calendario-proprio-dos-santos-brasil/)
+*/
```

Comment:
please put these in comments on the change itself, not at a file level

Reply:
Besides moving such comments to respective celebrations, use a proper (more conventional) syntax for multi-line comments, e.g. this comment would be formatted like this:

```suggestion
/*
 * Sources:
 *  (1) Palavra do Senhor III - Lecionário para as missas dos Santos, dos comuns,
 *      para diversas necessidades e votivas - Ed. Paulus (Lectionary for the Masses of Saints)
 *      Respecting Upper and LowerCase (https://archive.org/details/lecionario-santoral-indice)
 *  (2) Calendário Próprio do Brasil - CNBB (https://www.cnbb.org.br/missal-romano-calendario-proprio-dos-santos-brasil/)
 */
```

However, I’d use `1.` instead of `(1)`, as I would format comments as if they were in Markdown.

Reply:
That said, I am still not 100 % that the data should be sourced from cnbb.org.br. If I were you, I’d try to get access to the Portuguese Translation of the Roman Missal.

Reply:
Those files are the index of Oficial Liturgical Reading book, but only the index
https://archive.org/details/lecionario-santoral-indice
https://www.cnbb.org.br/missal-romano-calendario-proprio-dos-santos-brasil/

Reply:
Those are perfect sources! :pray: Use them primarily to provide source references please.

Ideally, I’d like more pictures taken from the Roman Missal (like the title page, title page verso [or colophon] with the book metadata, and additional pages with celebrations names and their format from the Proper of Time). However, this is more then enough to source all celebrations inscribed in the Brasilian Proper, both localisations and celebration metadata.

Reply:
I'll update those files with folder, edition...  when return to my city, those photos are taken by community friends tto help me.

Reply:
That would be awesome, thanks! :pray:

Later, I’ll create a list for you with what Roman Missal pages I/we are interested in.

Reply:
@RonaldoSCouto, here is the list of localisation data which can be sourced from the Roman Missal. I would like to get the three metadata pages. Thank you very much in advance! :pray:

<details>
   <summary>Roman Missal Source Pages</summary>

> ## Acronyms
>
> - GIRM: General Instruction or the Roman Missal;
> - RM: Roman Missal.
>
> # Pages and Data to source from the RM
>
> - metadata:
>   - front cover;
>   - title page;
>   - title page verso (or colophon);
> - General Roman Calendar (around p100 divided by months);
> - seasonal celebrations:
>   - `seasons.advent.privileged_weekday`;
>   - `seasons.advent.season`;
>   - `seasons.advent.sunday`;
>   - `seasons.advent.weekday`;
>   - `seasons.christmas_time.day`;
>   - `seasons.christmas_time.octave`;
>   - `seasons.christmas_time.season`;
>   - `seasons.christmas_time.second_sunday_after_christmas`;
>   - `seasons.christmas_time.after_epiphany`;
>   - `seasons.christmas_time.before_epiphany`;
>   - `seasons.lent.season`;
>   - `seasons.lent.day_after_ash_wed`;
>   - `seasons.lent.holy_week_day`;
>   - `seasons.lent.sunday`;
>   - `seasons.lent.weekday`;
>   - `seasons.easter_time.octave`;
>   - `seasons.easter_time.season`;
>   - `seasons.easter_time.sunday`;
>   - `seasons.easter_time.weekday`;
>   - `seasons.ordinary_time.season`;
>   - `seasons.ordinary_time.sunday`;
>   - `seasons.ordinary_time.weekday`;
>   - `seasons.paschal_triduum.season`;
>   - `periods.epiphany`;
>   - `periods.holy_week`;
> - cycles (I am not sure if you can find this in the RM apart from Proper of Time and Proper of Saints):
>   - `cycles.proper_of_time`;
>   - `cycles.proper_of_saints`;
>   - `cycles.sunday_year_a`;
>   - `cycles.sunday_year_b`;
>   - `cycles.sunday_year_c`;
>   - `cycles.weekday_year_1`;
>   - `cycles.weekday_year_2`;
>   - `cycles.psalter_week_1`;
>   - `cycles.psalter_week_2`;
>   - `cycles.psalter_week_3`;
>   - `cycles.psalter_week_4`;
> - colours (should be taken from GIRM no 346):
>   - `black`;
>   - `gold`;
>   - ??? `silver`;
>   - ??? `blue`;
>   - `green`;
>   - `purple`;
>   - `red`;
>   - `rose`;
>   - `white`;
> - keys from the Proper of Time:
>   - `ash_wednesday`;
>   - `divine_mercy_sunday`;
>   - `easter_sunday`;
>   - `friday_of_the_passion_of_the_lord`;
>   - `holy_family_of_jesus_mary_and_joseph`;
>   - `holy_saturday`;
>   - `holy_thursday`;
>   - `palm_sunday_of_the_passion_of_the_lord`;
>   - `pentecost_sunday`;
>   - `thursday_of_the_lords_supper`;
>   - `epiphany_of_the_lord`;
>   - `nativity_of_the_lord`;
>   - `mary_mother_of_god`;
>   - `baptism_of_the_lord`;
>   - `ash_wednesday`;
>   - `palm_sunday_of_the_passion_of_the_lord`;
>   - `holy_saturday`;
>   - `divine_mercy_sunday`;
>   - `ascension_of_the_lord`;
>   - `most_holy_body_and_blood_of_christ`;
>   - `most_holy_trinity`;
>   - `our_lord_jesus_christ_the_eternal_high_priest`;
>   - `most_sacred_heart_of_jesus`;
>   - `our_lord_jesus_christ_king_of_the_universe`;
> - ranks (should be taken from GIRM):
>   - `solemnity`;
>   - `sunday`;
>   - `feast`;
>   - `memorial`;
>   - `weekday`;
> - any keys from the Proper of Saints which can be sourced from the RM but are not listed in the General Roman Calendar (celebration list by listed months at around p100).

</details>